aether-ui/aether/ui/api/utils.py

```diff
@@ -661,3 +662,23 @@ def kernel_data_request(url='', method='get', data=None, headers=None):
 
 def wrap_kernel_headers(instance):
     return mt_utils.add_instance_realm_in_headers(instance, kernel_utils.get_kernel_auth_header())
+
+
+def delete_operation(url, data, object):
```

Comment:
`object` is a sort of reserved word (like `int`) could you rename the variable? The typical `obj` works ;-)